### PR TITLE
Feat: RTENU-81 Category data database

### PR DESCRIPTION
**Pull request by mehtis**
* API endpoints for getting custom fields in a page (/api/database/get-page-contents) and for getting files on a page (/api/alfresco/list-files)
* Database fields for linking frontend pages to alfresco
* Changed some scripts in package.json to use local in naming
* Refactored some existing CDK code to use more shared values (esp. for ENV)
* Refactored some backend code to get static, non-secret values from ENV instead of Parameter Store
* Added script for populating local db
* Added new fields to search queries
* Feat and local environments now set logging level to debug

### CODE REVIEW

File: packages/server/lambdas/alfresco/searchQueryBuilder/types.ts [hunk 1]
```diff
@@ -32,6 +33,10 @@ export type Paging = {
   skipCount: number;
 };
 
+export enum Include {
```
Comment (NinaDang97) [inline packages/server/lambdas/alfresco/searchQueryBuilder/types.ts:36]: "Include" name is a bit confusing. At least I don't understand what its purpose based on the name?

Comment (mehtis) [reply packages/server/lambdas/alfresco/searchQueryBuilder/types.ts:36]: That's what the field is called in the API, but it definitely is not descriptive. I'll think of something better. Thanks!

Comment (mehtis) [reply packages/server/lambdas/alfresco/searchQueryBuilder/types.ts]: Updated to AdditionalFields.

File: packages/server/prisma/schema.prisma [hunk 2]
```diff
@@ -11,6 +11,7 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+// Test models
```
Comment (saneDG) [inline packages/server/prisma/schema.prisma:14]: We should remove test models after this and start testing api/queries with "Real" models. This should be done in another PR where we also remove those deprecated lambdas and endpoints.

Comment (mehtis) [reply packages/server/prisma/schema.prisma:14]: Yup. Once we have tested this as working in main, we can start removing those.

File: packages/server/populate-local-db.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+set -a; source .env; set +a
+psql -h $LOCAL_IP -p $DB_PORT -d $DATABASE_NAME -U $DATABASE_USER << END_OF_SCRIPT
+TRUNCATE "CategoryDataContents", "CategoryDataBase" restart identity;
+
+INSERT INTO "CategoryDataBase" ("rataextraRequestPage", "alfrescoFolder", "writeRights")
+VALUES ('/linjakaaviot', '/123', 'Rataextra_kirjoitus_linjakaaviot');
+
+INSERT INTO "CategoryDataContents" ("baseId", "fields")
+VALUES (1, '{"test": "test"}')
+END_OF_SCRIPT
```
Comment (saneDG) [inline packages/server/populate-local-db.sh:11]: We could seed our database using TS [also](https://www.prisma.io/docs/guides/database/seed-database#example-seed-scripts).

But this approach works too 👍🏻

Comment (mehtis) [reply packages/server/populate-local-db.sh:11]: Definitely something to improve upon. And then there's the matter of where to store the dev-env values for population. Not sure if we want to have those in public like this, if they'd be populated from here.

**Review by saneDG — APPROVED** [review]

File: packages/server/lambdas/alfresco/searchQueryBuilder/__tests__/luceneQueryBuilder.test.ts [hunk 4]
```diff
@@ -126,23 +126,37 @@ describe('Lucene Query Builder', () => {
...
-  });
-  describe('lucenePagination', () => {
     it('should return given positive pagination page', () => {
       expect(luceneQueryBuilder.pagination(5)).toEqual({ maxItems: 10, skipCount: 5 });
     });
-  });
-
-  describe('lucenePagination', () => {
     it('should return zero given negative pagination page', () => {
       expect(luceneQueryBuilder.pagination(-4)).toEqual({ maxItems: 10, skipCount: 0 });
```
Comment (NinaDang97) [inline packages/server/lambdas/alfresco/searchQueryBuilder/__tests__/luceneQueryBuilder.test.ts:147]: Just a minor fix, could you change all `maxItems` to 25 in this test file? My previous PR increased the limit

File: packages/server/lambdas/alfresco/search.ts [hunk 5]
```diff
@@ -1,48 +1,30 @@
...
     const bodyRequest = searchQueryBuilder({
       searchParameters: parsedBody.searchParameters,
       page: parsedBody.page,
       language: parsedBody.language,
     });
-    log.info(bodyRequest, 'Complete body request');
+    log.debug(bodyRequest, 'Complete body request');
+
+    const alfrescoAPIUrl = getAlfrescoUrlBase();
+    const options = await getAlfrescoOptions(uid, { 'Content-Type': 'application/json;charset=UTF-8' });
```
Comment (NinaDang97) [inline packages/server/lambdas/alfresco/search.ts:23]: Thanks for refactoring!

File: packages/server/lambdas/alfresco/searchQueryBuilder/__tests__/index.test.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+describe('searchQueryBuilder', () => {
+  it("don't have additionalFields alongside query if not given", () => {
+    expect(searchQueryBuilder({ language: QueryLanguage.LUCENE, searchParameters: [], page: 0 })).toEqual({
+      query: {
+        language: 'lucene',
+        query: '',
+      },
+      paging: {
+        maxItems: 10,
```
Comment (NinaDang97) [inline packages/server/lambdas/alfresco/searchQueryBuilder/__tests__/index.test.ts:12]: It's 25 now :)

File: packages/server/lambdas/alfresco/list-files.ts [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    return response.data;
+  } catch (err) {
+    throw err;
+  }
+};
+
+const database = await DatabaseClient.build();
+
+let fileEndpointsCache: Array<CategoryDataBase> = [];
+
```
Comment (NinaDang97) [inline packages/server/lambdas/alfresco/list-files.ts:44]: Should we document by commenting in each `handleRequest` function what endpoint, method, body request structure,...? It'll be easier for team members to test

Comment (mehtis) [reply packages/server/lambdas/alfresco/list-files.ts:44]: This has been done now. Please share your thoughts!
